sample/src/main/res/layout/item_horizontal_list.xml

```diff
@@ -14,10 +14,16 @@
   ~ limitations under the License.
   -->
 
-<android.support.v7.widget.RecyclerView xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:id="@+id/post_list"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:paddingLeft="@dimen/normal_space"
-    android:clipToPadding="false"/>
+    android:descendantFocusability="blocksDescendants">
```

Comment:
According to the official documentation:
> android:descendantFocusability
Defines the relationship between the ViewGroup and its descendants when looking for a View to take focus.
Must be one of the following constant values.
blocksDescendants: The ViewGroup will block its descendants from receiving focus.

This means that `LinearLayout` will block its descendants from receiving focus. I did not understand why this would solve our current issue. Can you explain the reasons in detail? Thank you!

Chinese:
根据官方文档的描述，像你这么写之后，这个 `LinearLayout` 会优先获得并拦截 focus，这么做会不会影响子 items 的响应？并且看起来和解决我们的问题没有直接关系（虽然看起来解决了）。所以可以阐述或一起讨论下更详细的原因，谢谢！